CHANGES.rst

```diff
@@ -19,6 +19,8 @@ New Features
 
 - Snackbar history logger has been moved from an overlay to a separate tab in the right sidebar tray. [#3466]
 
+- App-level delete_subsets method to remove all subsets or subsets by label. [#3498]
```

Comment:
Is `app.delete_subsets` considered public API? If not, probably no need for change log.